app/component/SummaryPage.js

```diff
@@ -1679,14 +1679,11 @@ class SummaryPage extends React.Component {
       return null;
     }
     let filteredItineraries;
-    if (!detailView) {
```

Comment:
I can't think of any reason why this filter was only applied, when detailView was not selected. I leaving out the filter, itinerary indices get messed up.